Add specs for Updown::Check

FabienChaynes opened the pull request:
This PR adds specs for the `Updown::Check` class.
It uses `RSpec` as the test framework and `VCR` to record HTTP interactions.

I tested use cases described in the README so it actually kind of looks like integration tests I guess?
I'm not really sure about what would be the cleanest way to organize the files so I'm open to suggestion :slightly_smiling_face:

FabienChaynes committed:
Add specs for Updown::Check

askehansen commented on spec/updown/updown_check_spec.rb:
```diff
@@ -0,0 +1,127 @@
+    end
+  end
+
+  describe '#create' do
```
create is a class method so maybe it should be named ".create" instead of "#create"?

askehansen commented:
Nice work! It looks good, i just think we should change "#create" to ".create" :)

FabienChaynes commented:
You're definitely right, I amended the previous commit and changed "#create" to ".create" :slightly_smiling_face:
I also renamed the file from `updown_check_spec.rb` to `check_spec.rb`. I think it makes more sense because it's already in the `updown` folder.

askehansen commented:
Merged. Thanks!

jarthod commented:
Nice, thanks a lot @FabienChaynes!
